docs/source/receive-adapter.md

```diff
@@ -30,54 +31,56 @@ func main() {
 `adapter.Main` is responsible for:
 
 - Parsing the command flags and initializing the environment configuration
-returned by `youradapter.NewEnvConfig`.
+  returned by `youradapter.NewEnvConfig`.
 - Setting up the logger.
 - Setting up the profiler server (when enabled).
 - Setting up the metrics server.
 - Setting up tracing.
-- Setting up a ConfigMap watcher (when enabled).
-- Setting up a CloudEvent client, targeting a single sink, for single resource adapter,
-or otherwise targeting no sink.
-- Running *your adapter* in leader election mode (when enabled) for being
-highly-available and scalable.
+- Setting up a ConfigMap watcher (when enabled, multiple resources only).
```

Comment:
```suggestion
- Setting up a ConfigMap watcher (when enabled, always multiple resources).
```
Not sure if my suggestion makes sense, but "multiple resources only" is a bit confusing.

Reply:
Feel free to unhold or change this

Reply:
fixed. Thanks!